doc/cug.tex

```diff
@@ -3972,6 +3964,40 @@ \subsubsection{Trigger Types}
 \lstinline=$CYLC_DIR/examples/satellite/ext-triggers/suite.rc= is a working
 example of a simulated satellite processing suite.
 
+External triggers are not normally needed in date-time cycling suites driven
+by real time data that comes in at regular intervals.  In these cases a data
+retrieval task can be clock-triggered (and have appropriate retry intervals
+supplied) to submit at the expected data arrival time, so little time if any
+is wasted in polling.  For tasks that may have a highly variable arrival
+time, this configuration may be impractical and the external trigger
+mechanism supplies an attractive alternative.  If the incoming data
+are cycle-point-specific, the cycle point can be specified as part of the
+message:
+\lstset{language=suiterc}
+\begin{lstlisting}
+# suite "data-proc"
+[scheduling]
+    initial cycle point = 20150125T00
+    final cycle point   = 20150126T00
+    [[special tasks]]
+        external-trigger = variable-wait("data arrived for $CYLC_TASK_CYCLE_POINT")
+    [[dependencies]]
+        [[[T00]]]
+            graph = init-process => variable-wait => post-process
```

Comment:
I'd prefer just `get-data` here, as it is more obviously a "task name" than `variable-wait` (which might be confusing to some users, IMO).